### PR TITLE
[Merged by Bors] - Updates to tests and local testnet for Ganache 7 

### PR DESCRIPTION
**Pull request by realbigsean**
## Issue Addressed

#2961

## Proposed Changes

-- update `--chainId` -> `--chain.chainId`
-- remove `--keepAliveTimeout`
-- fix log to listen for
-- rename `ganache-cli` to `ganache` everywhere

### CODE REVIEW

**Review by paulhauner — APPROVED** [review]
LGTM!

**Comment by realbigsean** [comment]
Dang not sure why windows is failing, will have to check it out tomorrow

**Comment by realbigsean** [comment]
Strange cause it seems like ganache installed successfully on windows in this run https://github.com/sigp/lighthouse/actions/runs/1930473256

**Comment by realbigsean** [comment]
Attempting to match the deps used in the ganache githuba actions workflows: https://github.com/trufflesuite/ganache/blob/1d68fd03b8d03f129b9435a2cc1fcd6d270141a0/.github/workflows/pr.yml

**Comment by realbigsean** [comment]
No dice, trying this now: https://github.com/trufflesuite/ganache/blob/f2d1515a4eb26d9f326341cf3902ec09a04a8710/CONTRIBUTING.md#on-windows

File: .github/workflows/test-suite.yml [hunk 1]
```diff
@@ -50,8 +50,10 @@ jobs:
     - uses: actions/checkout@v1
     - name: Get latest version of stable Rust
       run: rustup update stable
-    - name: Install ganache-cli
-      run: npm install -g ganache-cli
+    - name: Install Windows Build Tools
+      run: npm install --global windows-build-tools
+    - name: Install ganache
+      run: npm install -g ganache
```
Comment (realbigsean) [inline .github/workflows/test-suite.yml]:
```suggestion
    - name: Use Python 2.7
       uses: actions/setup-python@v2
       with:
         python-version: "2.7"
```

File: .github/workflows/test-suite.yml [hunk 2]
```diff
@@ -50,8 +50,10 @@ jobs:
     - uses: actions/checkout@v1
     - name: Get latest version of stable Rust
       run: rustup update stable
-    - name: Install ganache-cli
-      run: npm install -g ganache-cli
+    - name: Use Python 2.7
+       uses: actions/setup-python@v2
+       with:
+         python-version: "2.7"
```
Comment (realbigsean) [inline .github/workflows/test-suite.yml]:
```suggestion
    - name: Use Python 2.7
      uses: actions/setup-python@v2
      with:
        python-version: "2.7"
```

File: .github/workflows/test-suite.yml [hunk 3]
```diff
@@ -50,8 +50,10 @@ jobs:
     - uses: actions/checkout@v1
     - name: Get latest version of stable Rust
       run: rustup update stable
-    - name: Install ganache-cli
-      run: npm install -g ganache-cli
+    - name: Use Python 2.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: "2.7"
     - name: Install make
```
Comment (realbigsean) [inline .github/workflows/test-suite.yml]:
```suggestion
        python-version: "2.7"
    - name: Install ganache
      run: npm install -g ganache
    - name: Install make
```

File: .github/workflows/test-suite.yml [hunk 4]
```diff
@@ -50,8 +50,12 @@ jobs:
     - uses: actions/checkout@v1
     - name: Get latest version of stable Rust
       run: rustup update stable
-    - name: Install ganache-cli
-      run: npm install -g ganache-cli
+    - name: Use Python 2.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: "2.7"
+    - name: Install ganache
```
Comment (realbigsean) [inline .github/workflows/test-suite.yml:63]:
```suggestion
    - name: Set node config to use python2.7
      run: npm config set python python2.7
    - name: Set msvs_version to 2019
      run: npm config set msvs_version 2019 --global
    - name: Install ganache
```

File: .github/workflows/test-suite.yml [hunk 5]
```diff
@@ -50,8 +50,16 @@ jobs:
     - uses: actions/checkout@v1
     - name: Get latest version of stable Rust
       run: rustup update stable
-    - name: Install ganache-cli
-      run: npm install -g ganache-cli
+    - name: Use Python 2.7
```
Comment (realbigsean) [inline .github/workflows/test-suite.yml:55]:
```suggestion
    - name: Add msbuild to PATH
      uses: microsoft/setup-msbuild@v1.0.2
    - name: Use Python 2.7
```

File: .github/workflows/test-suite.yml [hunk 6]
```diff
@@ -50,8 +50,18 @@ jobs:
...
-    - name: Install ganache-cli
-      run: npm install -g ganache-cli
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.0.2
+    - name: Use Python 2.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: "2.7"
+    - name: Set node config to use python2.7
+      run: npm config set python python2.7
```
Comment (realbigsean) [inline .github/workflows/test-suite.yml]:
```suggestion
      run: npm config set python %pythonLocation%
```

**Comment by realbigsean** [comment]
Ok hopefully this works on bors now. I think the issue was the version of python being used in the build process on windows. I've marked this as `backward-incompat` because tests and the local simulator will now break if ganache V7 is not installed. But there's nothing outside of tests/local testnet affected.

**Review by michaelsproul — APPROVED** [review]
LGTM let's try this again

**Pull request closed without merging**